bin/test-image

```diff
@@ -72,8 +72,14 @@ if [ "${TYPE}" = "vm" ]; then
     lxc init "${TEST_IMAGE}" "${TEST_IMAGE}" \
         --vm \
         -c limits.cpu=4 \
-        -c limits.memory=4GiB \
-        -c security.secureboot=false
+        -c limits.memory=4GiB
+
+    # Some distros don't support secure boot.
+    case "${DISTRO}" in
+        alpine|archlinux|gentoo|nixos)
+            lxc config set "${TEST_IMAGE}" security.secureboot=true
```

Comment:
shouldnt this be false?

Whats with the `)` at the end but no starting bracket, is this some weird bash-ism?

Reply:
Oh, indeed, `false` it should have been. Fixed, thanks.

As for the `)` missing the opening one, it is not a bashism but now that you call it out it looks weird. `man dash`:
```
The syntax of the case command is

       case word in
       [(]pattern) list ;;
       ...
       esac

The pattern can actually be one or more patterns (see Shell Patterns described later), separated by “|” characters.
The “(” character before the pattern is optional.
```
I don't remember seeing `case` with an opening `(` though. I'm find introducing it if that reads better for you.

Reply:
Ah ok thanks, from my PHP, Java, Go, C eyes thats a definite code smell, but seems like its ok in shell.